provisioner/node_affinity.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2020 The OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package provisioner
+
+import (
+	"os"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+var (
+	nodeAffinityKey = "NODEAFFINITY"
+)
+
+// getEnv fetches the provided environment variable's value
+func getEnv(envKey string) (value string) {
```

Comment:
better to move it to `provisioner/env.go` file

Reply:
Updated PTAL